{"module": "/app/src/utils/config.py", "path": "/app/config/config.yaml", "event": "YAML configuration loaded", "timestamp": "2025-05-10T01:02:38.278418Z", "level": "info"}
{"module": "/app/src/utils/config.py", "path": "/app/config/config.yaml", "event": "YAML configuration loaded", "timestamp": "2025-05-10T01:28:16.315214Z", "level": "info"}
{"module": "/app/src/utils/config.py", "path": "/app/config/config.yaml", "event": "YAML configuration loaded", "timestamp": "2025-05-10T01:32:02.046448Z", "level": "info"}
{"module": "/app/src/utils/config.py", "path": "/app/config/config.yaml", "event": "YAML configuration loaded", "timestamp": "2025-05-10T01:35:15.561246Z", "level": "info"}
{"module": "/app/src/utils/db_connection.py", "variable": "POSTGRES_DATABASE_URL", "event": "Missing environment variable", "timestamp": "2025-05-10T01:35:17.072063Z", "level": "error"}
{"module": "/app/src/utils/config.py", "path": "/app/config/config.yaml", "event": "YAML configuration loaded", "timestamp": "2025-05-10T01:49:03.203821Z", "level": "info"}
{"module": "/app/src/utils/db_connection.py", "variable": "POSTGRES_DATABASE_URL", "event": "Missing environment variable", "timestamp": "2025-05-10T01:49:04.100483Z", "level": "error"}
{"module": "/app/src/utils/config.py", "path": "/app/config/config.yaml", "event": "YAML configuration loaded", "timestamp": "2025-05-10T01:55:13.580719Z", "level": "info"}
{"module": "/app/src/utils/db_connection.py", "variable": "POSTGRES_DATABASE_URL", "event": "Missing environment variable", "timestamp": "2025-05-10T01:55:14.397752Z", "level": "error"}
{"module": "/app/src/utils/config.py", "path": "/app/config/config.yaml", "event": "YAML configuration loaded", "timestamp": "2025-05-10T02:09:24.190106Z", "level": "info"}
{"module": "/app/src/utils/db_connection.py", "event": "Database connection pool established", "timestamp": "2025-05-10T02:09:28.303504Z", "level": "info"}
{"module": "/app/src/utils/config.py", "path": "/app/config/config.yaml", "event": "YAML configuration loaded", "timestamp": "2025-05-10T02:31:44.871645Z", "level": "info"}
{"module": "/app/src/utils/db_connection.py", "event": "Database connection pool established", "timestamp": "2025-05-10T02:31:46.370950Z", "level": "info"}
{"module": "utils", "path": "/app/config/config.yaml", "event": "Configuration loaded", "timestamp": "2025-05-10T02:31:46.377986Z", "level": "info"}
{"module": "/app/src/utils/config.py", "path": "/app/config/config.yaml", "event": "YAML configuration loaded", "timestamp": "2025-05-10T23:28:08.525976Z", "level": "info"}
{"module": "/app/src/utils/db_connection.py", "event": "Database connection pool established", "timestamp": "2025-05-10T23:28:10.530904Z", "level": "info"}
{"module": "utils", "path": "/app/config/config.yaml", "event": "Configuration loaded", "timestamp": "2025-05-10T23:28:10.548826Z", "level": "info"}
{"module": "/app/src/main.py", "event": "ETL Process Started", "timestamp": "2025-05-10T23:28:10.563947Z", "level": "info"}
{"module": "extract", "event": "Starting data extraction...", "timestamp": "2025-05-10T23:28:10.564435Z", "level": "info"}
{"module": "/app/src/utils/config.py", "event": "Loading .env file", "timestamp": "2025-05-10T23:28:10.564883Z", "level": "info"}
{"module": "/app/src/utils/config.py", "variable": "API_KEY", "event": "Environment variable not set", "timestamp": "2025-05-10T23:28:10.568674Z", "level": "warning"}
{"module": "/app/src/utils/api_requests.py", "url": "https://www.alphavantage.co/query?function=TIME_SERIES_INTRADAY&symbol=IBM&interval=5min&adjusted=false&apikey=None", "timeout": 30, "event": "Sending request", "timestamp": "2025-05-10T23:28:10.569110Z", "level": "info"}
{"module": "/app/src/utils/api_requests.py", "status_code": 200, "event": "Request successful", "timestamp": "2025-05-10T23:28:11.213694Z", "level": "info"}
{"module": "utils", "event": "No API errors found", "timestamp": "2025-05-10T23:28:11.214991Z", "level": "info"}
{"module": "utils", "event": "Data validated successfully", "timestamp": "2025-05-10T23:28:11.215698Z", "level": "info"}
{"module": "extract", "event": "An unexpected error occurred during extraction.", "timestamp": "2025-05-10T23:28:11.234930Z", "level": "error", "exception": "Traceback (most recent call last):\n  File \"/app/src/extract.py\", line 87, in extract_data\n    raw_data_dir.mkdir(parents=True, exist_ok=True)\n  File \"/usr/local/lib/python3.9/pathlib.py\", line 1323, in mkdir\n    self._accessor.mkdir(self, mode)\nFileExistsError: [Errno 17] File exists: '/app/data/raw_data'"}
{"module": "/app/src/main.py", "event": "Data extraction failed. ETL process terminated.", "timestamp": "2025-05-10T23:28:11.239381Z", "level": "error"}
{"module": "/app/src/utils/config.py", "path": "/app/config/config.yaml", "event": "YAML configuration loaded", "timestamp": "2025-05-11T00:06:38.937780Z", "level": "info"}
{"module": "/app/src/utils/db_connection.py", "event": "Database connection pool established", "timestamp": "2025-05-11T00:06:40.381978Z", "level": "info"}
{"module": "utils", "path": "/app/config/config.yaml", "event": "Configuration loaded", "timestamp": "2025-05-11T00:06:40.410469Z", "level": "info"}
{"module": "/app/src/main.py", "event": "ETL Process Started", "timestamp": "2025-05-11T00:06:40.434091Z", "level": "info"}
{"module": "extract", "event": "Starting data extraction...", "timestamp": "2025-05-11T00:06:40.434789Z", "level": "info"}
{"module": "/app/src/utils/config.py", "event": "Loading .env file", "timestamp": "2025-05-11T00:06:40.435340Z", "level": "info"}
{"module": "/app/src/utils/config.py", "variable": "API_KEY", "event": "Environment variable not set", "timestamp": "2025-05-11T00:06:40.441255Z", "level": "warning"}
{"module": "/app/src/utils/api_requests.py", "url": "https://www.alphavantage.co/query?function=TIME_SERIES_INTRADAY&symbol=IBM&interval=5min&adjusted=false&apikey=None", "timeout": 30, "event": "Sending request", "timestamp": "2025-05-11T00:06:40.441976Z", "level": "info"}
{"module": "/app/src/utils/api_requests.py", "status_code": 200, "event": "Request successful", "timestamp": "2025-05-11T00:06:40.692347Z", "level": "info"}
{"module": "utils", "event": "No API errors found", "timestamp": "2025-05-11T00:06:40.697791Z", "level": "info"}
{"module": "utils", "event": "Data validated successfully", "timestamp": "2025-05-11T00:06:40.700040Z", "level": "info"}
{"module": "extract", "event": "An unexpected error occurred during extraction.", "timestamp": "2025-05-11T00:06:40.709239Z", "level": "error", "exception": "Traceback (most recent call last):\n  File \"/app/src/extract.py\", line 87, in extract_data\n    raw_data_dir.mkdir(parents=True, exist_ok=True)\n  File \"/usr/local/lib/python3.9/pathlib.py\", line 1323, in mkdir\n    self._accessor.mkdir(self, mode)\nFileExistsError: [Errno 17] File exists: '/app/data/raw_data'"}
{"module": "/app/src/main.py", "event": "Data extraction failed. ETL process terminated.", "timestamp": "2025-05-11T00:06:40.716882Z", "level": "error"}
{"module": "/app/src/utils/config.py", "path": "/app/config/config.yaml", "event": "YAML configuration loaded", "timestamp": "2025-05-11T00:11:45.298008Z", "level": "info"}
{"module": "/app/src/utils/db_connection.py", "event": "Database connection pool established", "timestamp": "2025-05-11T00:11:48.160100Z", "level": "info"}
{"module": "utils", "path": "/app/config/config.yaml", "event": "Configuration loaded", "timestamp": "2025-05-11T00:11:48.176816Z", "level": "info"}
{"module": "/app/src/main.py", "event": "ETL Process Started", "timestamp": "2025-05-11T00:11:48.198534Z", "level": "info"}
{"module": "extract", "event": "Starting data extraction...", "timestamp": "2025-05-11T00:11:48.199244Z", "level": "info"}
{"module": "/app/src/utils/config.py", "event": "Loading .env file", "timestamp": "2025-05-11T00:11:48.199906Z", "level": "info"}
{"module": "/app/src/utils/config.py", "variable": "API_KEY", "event": "Environment variable not set", "timestamp": "2025-05-11T00:11:48.207623Z", "level": "warning"}
{"module": "/app/src/utils/api_requests.py", "url": "https://www.alphavantage.co/query?function=TIME_SERIES_INTRADAY&symbol=IBM&interval=5min&adjusted=false&apikey=None", "timeout": 30, "event": "Sending request", "timestamp": "2025-05-11T00:11:48.208381Z", "level": "info"}
{"module": "/app/src/utils/api_requests.py", "status_code": 200, "event": "Request successful", "timestamp": "2025-05-11T00:11:48.505829Z", "level": "info"}
{"module": "utils", "event": "No API errors found", "timestamp": "2025-05-11T00:11:48.506838Z", "level": "info"}
{"module": "utils", "event": "Data validated successfully", "timestamp": "2025-05-11T00:11:48.507319Z", "level": "info"}
{"module": "extract", "event": "An unexpected error occurred during extraction.", "timestamp": "2025-05-11T00:11:48.514740Z", "level": "error", "exception": "Traceback (most recent call last):\n  File \"/app/src/extract.py\", line 87, in extract_data\n    raw_data_dir.mkdir(parents=True, exist_ok=True)\n  File \"/usr/local/lib/python3.9/pathlib.py\", line 1323, in mkdir\n    self._accessor.mkdir(self, mode)\nFileExistsError: [Errno 17] File exists: '/app/data/raw_data'"}
{"module": "/app/src/main.py", "event": "Data extraction failed. ETL process terminated.", "timestamp": "2025-05-11T00:11:48.521258Z", "level": "error"}
{"module": "/app/src/utils/config.py", "path": "/app/config/config.yaml", "event": "YAML configuration loaded", "timestamp": "2025-05-11T00:29:44.637864Z", "level": "info"}
{"module": "/app/src/utils/db_connection.py", "event": "Database connection pool established", "timestamp": "2025-05-11T00:29:45.003401Z", "level": "info"}
{"module": "utils", "path": "/app/config/config.yaml", "event": "Configuration loaded", "timestamp": "2025-05-11T00:29:45.018005Z", "level": "info"}
{"module": "/app/src/main.py", "event": "ETL Process Started", "timestamp": "2025-05-11T00:29:45.031539Z", "level": "info"}
{"module": "extract", "event": "Starting data extraction...", "timestamp": "2025-05-11T00:29:45.032140Z", "level": "info"}
{"module": "/app/src/utils/config.py", "event": "Loading .env file", "timestamp": "2025-05-11T00:29:45.032542Z", "level": "info"}
{"module": "/app/src/utils/config.py", "variable": "API_KEY", "event": "Environment variable not set", "timestamp": "2025-05-11T00:29:45.035715Z", "level": "warning"}
{"module": "/app/src/utils/api_requests.py", "url": "https://www.alphavantage.co/query?function=TIME_SERIES_INTRADAY&symbol=IBM&interval=5min&adjusted=false&apikey=None", "timeout": 30, "event": "Sending request", "timestamp": "2025-05-11T00:29:45.036119Z", "level": "info"}
{"module": "/app/src/utils/api_requests.py", "status_code": 200, "event": "Request successful", "timestamp": "2025-05-11T00:29:45.520139Z", "level": "info"}
{"module": "utils", "event": "No API errors found", "timestamp": "2025-05-11T00:29:45.521889Z", "level": "info"}
{"module": "utils", "event": "Data validated successfully", "timestamp": "2025-05-11T00:29:45.522780Z", "level": "info"}
{"module": "extract", "event": "An unexpected error occurred during extraction.", "timestamp": "2025-05-11T00:29:45.528690Z", "level": "error", "exception": "Traceback (most recent call last):\n  File \"/app/src/extract.py\", line 87, in extract_data\n    raw_data_dir.mkdir(parents=True, exist_ok=True)\n  File \"/usr/local/lib/python3.9/pathlib.py\", line 1323, in mkdir\n    self._accessor.mkdir(self, mode)\nFileExistsError: [Errno 17] File exists: '/app/data/raw_data'"}
{"module": "/app/src/main.py", "event": "Data extraction failed. ETL process terminated.", "timestamp": "2025-05-11T00:29:45.533168Z", "level": "error"}
{"module": "/app/src/utils/config.py", "path": "/app/config/config.yaml", "event": "YAML configuration loaded", "timestamp": "2025-05-11T00:35:31.095438Z", "level": "info"}
{"module": "/app/src/utils/db_connection.py", "event": "Database connection pool established", "timestamp": "2025-05-11T00:35:31.495272Z", "level": "info"}
{"module": "utils", "path": "/app/config/config.yaml", "event": "Configuration loaded", "timestamp": "2025-05-11T00:35:31.510697Z", "level": "info"}
{"module": "/app/src/main.py", "event": "ETL Process Started", "timestamp": "2025-05-11T00:35:31.524400Z", "level": "info"}
{"module": "extract", "event": "Starting data extraction...", "timestamp": "2025-05-11T00:35:31.524799Z", "level": "info"}
{"module": "/app/src/utils/config.py", "event": "Loading .env file", "timestamp": "2025-05-11T00:35:31.525155Z", "level": "info"}
{"module": "/app/src/utils/config.py", "variable": "API_KEY", "event": "Environment variable not set", "timestamp": "2025-05-11T00:35:31.527991Z", "level": "warning"}
{"module": "/app/src/utils/api_requests.py", "url": "https://www.alphavantage.co/query?function=TIME_SERIES_INTRADAY&symbol=IBM&interval=5min&adjusted=false&apikey=None", "timeout": 30, "event": "Sending request", "timestamp": "2025-05-11T00:35:31.528321Z", "level": "info"}
{"module": "/app/src/utils/api_requests.py", "status_code": 200, "event": "Request successful", "timestamp": "2025-05-11T00:35:31.999420Z", "level": "info"}
{"module": "utils", "event": "No API errors found", "timestamp": "2025-05-11T00:35:32.001328Z", "level": "info"}
{"module": "utils", "event": "Data validated successfully", "timestamp": "2025-05-11T00:35:32.002319Z", "level": "info"}
{"module": "extract", "event": "An unexpected error occurred during extraction.", "timestamp": "2025-05-11T00:35:32.007846Z", "level": "error", "exception": "Traceback (most recent call last):\n  File \"/app/src/extract.py\", line 87, in extract_data\n    raw_data_dir.mkdir(parents=True, exist_ok=True)\n  File \"/usr/local/lib/python3.9/pathlib.py\", line 1323, in mkdir\n    self._accessor.mkdir(self, mode)\nFileExistsError: [Errno 17] File exists: '/app/data/raw_data'"}
{"module": "/app/src/main.py", "event": "Data extraction failed. ETL process terminated.", "timestamp": "2025-05-11T00:35:32.018263Z", "level": "error"}
{"module": "/app/src/utils/config.py", "path": "/app/config/config.yaml", "event": "YAML configuration loaded", "timestamp": "2025-05-11T00:42:34.844294Z", "level": "info"}
{"module": "/app/src/utils/db_connection.py", "event": "Database connection pool established", "timestamp": "2025-05-11T00:42:36.023184Z", "level": "info"}
{"module": "utils", "path": "/app/config/config.yaml", "event": "Configuration loaded", "timestamp": "2025-05-11T00:42:36.052596Z", "level": "info"}
{"module": "/app/src/main.py", "event": "ETL Process Started", "timestamp": "2025-05-11T00:42:36.069449Z", "level": "info"}
{"module": "extract", "event": "Starting data extraction...", "timestamp": "2025-05-11T00:42:36.070018Z", "level": "info"}
{"module": "/app/src/utils/config.py", "event": "Loading .env file", "timestamp": "2025-05-11T00:42:36.070447Z", "level": "info"}
{"module": "/app/src/utils/config.py", "variable": "API_KEY", "event": "Environment variable not set", "timestamp": "2025-05-11T00:42:36.074150Z", "level": "warning"}
{"module": "/app/src/utils/api_requests.py", "url": "https://www.alphavantage.co/query?function=TIME_SERIES_INTRADAY&symbol=IBM&interval=5min&adjusted=false&apikey=None", "timeout": 30, "event": "Sending request", "timestamp": "2025-05-11T00:42:36.074628Z", "level": "info"}
{"module": "/app/src/utils/api_requests.py", "status_code": 200, "event": "Request successful", "timestamp": "2025-05-11T00:42:36.820965Z", "level": "info"}
{"module": "utils", "event": "No API errors found", "timestamp": "2025-05-11T00:42:36.822144Z", "level": "info"}
{"module": "utils", "event": "Data validated successfully", "timestamp": "2025-05-11T00:42:36.822687Z", "level": "info"}
{"module": "utils", "file_path": "PosixPath('/app/data/raw_data/data_20250511_004236.json')", "event": "Data saved", "timestamp": "2025-05-11T00:42:36.835620Z", "level": "info"}
{"module": "extract", "path": "/app/data/raw_data/data_20250511_004236.json", "event": "Data extracted and saved successfully", "timestamp": "2025-05-11T00:42:36.836609Z", "level": "info"}
{"module": "utils", "path": "/app/config/config.yaml", "event": "Configuration loaded", "timestamp": "2025-05-11T00:42:36.845185Z", "level": "info"}
{"module": "/app/src/utils/config.py", "path": "/app/config/config.yaml", "event": "YAML configuration loaded", "timestamp": "2025-05-11T01:20:40.987536Z", "level": "info"}
{"module": "/app/src/utils/db_connection.py", "event": "Database connection pool established", "timestamp": "2025-05-11T01:20:42.222319Z", "level": "info"}
{"module": "utils", "path": "/app/config/config.yaml", "event": "Configuration loaded", "timestamp": "2025-05-11T01:20:42.238432Z", "level": "info"}
{"module": "/app/src/main.py", "event": "ETL Process Started", "timestamp": "2025-05-11T01:20:42.252380Z", "level": "info"}
{"module": "extract", "event": "Starting data extraction...", "timestamp": "2025-05-11T01:20:42.253019Z", "level": "info"}
{"module": "/app/src/utils/config.py", "event": "Loading .env file", "timestamp": "2025-05-11T01:20:42.253453Z", "level": "info"}
{"module": "/app/src/utils/config.py", "variable": "API_KEY", "event": "Environment variable not set", "timestamp": "2025-05-11T01:20:42.257212Z", "level": "warning"}
{"module": "/app/src/utils/api_requests.py", "url": "https://www.alphavantage.co/query?function=TIME_SERIES_INTRADAY&symbol=IBM&interval=5min&adjusted=false&apikey=None", "timeout": 30, "event": "Sending request", "timestamp": "2025-05-11T01:20:42.257754Z", "level": "info"}
{"module": "/app/src/utils/api_requests.py", "status_code": 200, "event": "Request successful", "timestamp": "2025-05-11T01:20:42.740247Z", "level": "info"}
{"module": "utils", "event": "No API errors found", "timestamp": "2025-05-11T01:20:42.741790Z", "level": "info"}
{"module": "utils", "event": "Data validated successfully", "timestamp": "2025-05-11T01:20:42.742468Z", "level": "info"}
{"module": "utils", "file_path": "PosixPath('/app/data/raw_data/data_20250511_012042.json')", "event": "Data saved", "timestamp": "2025-05-11T01:20:42.751969Z", "level": "info"}
{"module": "extract", "path": "/app/data/raw_data/data_20250511_012042.json", "event": "Data extracted and saved successfully", "timestamp": "2025-05-11T01:20:42.752730Z", "level": "info"}
{"module": "utils", "path": "/app/config/config.yaml", "event": "Configuration loaded", "timestamp": "2025-05-11T01:20:42.763715Z", "level": "info"}
